app/styles/global_styles.js

```diff
@@ -8,7 +8,7 @@ body {
     height: 100%;
     margin: 0px;
     background-color: #F5F5F5;
-    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-family: 'Ubuntu', sans-serif;
```

Comment:
My dream is the styles that are going to change each event should be separated from these standard styles for clarity and easy updating